alacritty/src/event.rs

```diff
@@ -1481,6 +1481,9 @@ impl Processor {
                 return;
             }
 
+            // NOTE when moving event to be handled without batching, the `dirty` must be changed
+            // to `display.window.request_redraw`, otherwise we could have a situation with an
+            // empty queue, but dirty bit being set.
```

Comment:
I'm not sure whether I should comment like that, or add a special `catch-all` in `AboutToWait` when queue is empty, but we have a dirty bit set. Probably better to
have a catch-all in `AboutToWait`?